.github/workflows/ci-cd-workflow.yml

```diff
@@ -39,16 +38,7 @@ jobs:
       # treat warnings as errors (-W)...
       # ...but not when being nitpicky (-n)
       run: |
-          sphinx-build -M html docs/source docs/build -qW
-          sphinx-build -M html docs/source docs/build -Eqn -b coverage
-          if [[ -s docs/build/html/python.txt ]]
-          then
-              echo
-              echo \"Error: Documentation missing:\"
-              echo
-              cat docs/build/html/python.txt
-              exit 1
-          fi
+          sphinx-build -M html docs/source docs/build -Evn
```

Comment:
I had to drop the -W flag as there are a couple of typing related warnings that aren't our fault, but I can't suppress

Reply:
Fair. Make an issue so we don't forget then let's resolve here as that will be a headache to fix?